packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -1519,5 +1547,34 @@ export function makeWallet({
       .then(addInviteDepositFacet);
     zoeInvitePurse = wallet.getPurse(ZOE_INVITE_PURSE_PETNAME);
   };
-  return { admin: wallet, initialized: initialize() };
+
+  // Importing assets as virtual purses from the bank is a highly-trusted path.
+  // We don't want to expose this mechanism to the user, in case they shoot
+  // themselves in the foot with it by importing an asset/virtual purse they
+  // don't really trust.
+  const importBankAssets = async bank => {
+    observeIteration(E(bank).getAssetSubscription(), {
+      async updateState({ proposedName, issuerName, issuer, brand }) {
+        try {
+          await addIssuer(issuerName, issuer);
+          const purse = await E(bank).getPurse(brand);
+          // We can import this purse, because we trust the bank.
+          await internalUnsafeImportPurse(
+            issuerName,
+            proposedName,
+            true,
+            () => purse,
```

Comment:
I think this function is unnecessary and we can just pass the purse directly in both this case and the normal case.

Reply:
I will pass the purse or falsy if we want to make an empty purse.  We have to handle the empty purse case specially because we don't know the issuer when we do `makeEmptyPurse(...)`.